src/super_gradients/common/sg_loggers/deci_platform_sg_logger.py

```diff
@@ -74,6 +75,18 @@ def upload(self):
         self._upload_latest_file_starting_with(start_with=CONSOLE_LOGS_PREFIX)
         self._upload_folder_files(folder_name=".hydra")
 
+        if self.upload_model:
```

Comment:
This implementation will upload all pth files over and over again.
That is redundant and will take a lot of networks capacity.
Instead, override `add_checkpoint` function. This way, you will upload only new files.

Reply:
Changed in https://github.com/Deci-AI/super-gradients/pull/799/commits/386678f10d0d2d85e0065ee4b7c095f8a86699be